src/main/kotlin/blackjack/application/BlackJackService.kt

```diff
@@ -12,23 +12,31 @@ private const val FIRST_DRAW_COUNT = 2
 object BlackJackService {
 
     fun createPlayers(names: List<String>): List<Player> {
```

Comment:
```suggestion
    fun createPlayers(names: List<String>): Players {
```

일급 컬렉션으로 만들어서 개선 해볼까요?